README.md

```diff
@@ -2,15 +2,14 @@
 
 ## Prerequisites
 
-If you want to use a preconfigured environment, you only need to launch one of the following virtual machine providers.
+Use the button below to launch a preconfigured environment in your web browser or Visual Studio Code.
 
-[![Open with Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/blab/veme-2022/HEAD)
-[![Open with Gitpod](https://img.shields.io/badge/Open%20with-Gitpod-908a85?logo=gitpod)](https://gitpod.io/#https://github.com/blab/veme-2022.git)
+[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/nextstrain/veme-2022)
 
 If you want to use your local environment on your personal computer, you will need to:
 
   - [Install Nextstrain](https://docs.nextstrain.org/en/latest/install.html) on your computer or launch one of the virtual machine providers listed above.
-  - Clone this repository with `git clone https://github.com/blab/veme-2022.git` and change into the resulting `veme-2022` directory.
+  - Clone this repository with `git clone https://github.com/nextstrain/veme-2022.git` and change into the resulting `veme-2022` directory.
```

Comment:
This is probably a bigger discussion than is appropriate for the bounds of this PR, but: perhaps we should consider renaming the repo to something more descriptive, more generic, and less dated looking?

Reply:
The scope of this PR is intentionally broad as "update everything", so I think good to discuss here. My proposed rename is "nextstrain-walkthrough" with the same reasoning as in 6fd86ec1af527fa3dec8cbaab24e0bec335a561b.

Reply:
+1 for `nextstrain-walkthrough` ...but i will see your bikeshed and raise you `nextstrain-tour`?

Reply:
"walkthru"? 😆 maybe after using it 100 times it'll feel normal like "drive thru"...

Reply:
I was thinking that too, but as more than just a direct replacement:

- "Running a phylogenetic workflow" shouldn't point to this with the existing phrasing of "learn how to create the workflow in this tutorial", unless we want to replace the underlying data and workflow used in that tutorial.
- I'd like to reconsider the name "creating a phylogenetic workflow", at least by dropping the term "workflow" since that's beyond the scope.

Reply:
I've renamed as `nextstrain-walkthrough` for now. While it's not perfect, I think it's at least more self-descriptive than `veme-2022` and will be useful for initial sharing of the draft.

Reply:
Continuing conversation in https://github.com/nextstrain/nextstrain-walkthrough/issues/9